starboard/build/config/modular/BUILD.gn

```diff
@@ -208,3 +208,20 @@ config("no_pedantic_warnings") {
     "-Wno-undefined-var-template",
   ]
 }
+
+# These are specifically for modular builds of Cobalt shared libraries.
+# They wrap underlying POSIX implementations to be cross-compatible with
+# musl-based types that are used in modular builds. Evergreen builds use
```

Comment:
How is it that the posix and musl implementation don't match (if I'm understanding this comment correctly)? And why is it that modular is different than monolithic in this case?